docs/guides/introduction.md

```diff
@@ -11,11 +11,12 @@ head:
 
 **WebContainers are a browser-based runtime for executing Node.js applications and operating system commands, entirely inside your browser tab.** Apps that previously required cloud VMs to execute user code, in WebContainers can run entirely client-side with [a number of benefits over the legacy cloud VM](#webcontainers-versus-cloud-vm-approach).
 
-**WebContainer API is perfect for interactive coding experiences.** Among its most common use cases are production-grade IDEs, programming tutorials, next-generation documentation, AI applications, or employee onboarding platforms. WebContainers have been [battle-tested by millions of users](#who-s-using-webcontainers) of StackBlitz classic editor, Codeflow, the official SvelteKit tutorial, and Cloudflare Wrangler workers, among others.
+**WebContainer API is perfect for interactive coding experiences.** Among its most common use cases are AI applications, adding in-browser code execution to your existing product, programming tutorials, next-generation documentation, browser based IDEs, and employee onboarding platforms. WebContainers have been [battle-tested by millions of users](#who-s-using-webcontainers) of StackBlitz, and inside the interactive coding environments built by [Svelte](https://learn.svelte.dev/tutorial/welcome-to-svelte), [Angular](https://angular.dev/tutorials/first-app), and [Nuxt](https://learn-dev.nuxt.com/) among others.
 
-Ready to try it? Check out this WebContainer API starter in an editor running on... WebContainers as well!
 
-[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://webcontainer.new)
+:::tip Ready to try it out for yourself?
+Get started in minutes with [our Quickstart guide here](./quickstart).
```

Comment:
I like the redirection to the quickstart but I don't think we should give up the instant demo (our core strength). Minutes is ages for giving a StackBlitz project a spin.

```suggestion
Check out this [WebContainer API starter](https://webcontainer.new/) or see [our Quickstart guide](./quickstart) to get familiar with what's possible!
```